src/yunohost/tools.py

```diff
@@ -265,21 +265,31 @@ def tools_postinstall(domain, password, ignore_dyndns=False):
     # Create SSL CA
     service_regen_conf(['ssl'], force=True)
     ssl_dir = '/usr/share/yunohost/yunohost-config/ssl/yunoCA'
-    command_list = [
+    commands = [
         'echo "01" > %s/serial' % ssl_dir,
         'rm %s/index.txt'       % ssl_dir,
         'touch %s/index.txt'    % ssl_dir,
         'cp %s/openssl.cnf %s/openssl.ca.cnf' % (ssl_dir, ssl_dir),
-        'sed -i "s/yunohost.org/%s/g" %s/openssl.ca.cnf ' % (domain, ssl_dir),
+        'sed -i s/yunohost.org/%s/g %s/openssl.ca.cnf ' % (domain, ssl_dir),
```

Comment:
It's been a long time, but I think it's because of the difference commands are executed with os.system compared to subprocess. os.system() is closer to what you would really type in a real-life shell, so you can put quotes which are then forgotten, because they are only here to be able to have, for instance, arguments with space in it. 

With subprocess, you directly feed a list and therefore don't need quotes. And in fact , if you put quote, they'll stay there (as if you escaped them, e.g. typing `sed -i \"s/foo/bar/g\"`).